api/buf.yaml

```diff
@@ -12,21 +12,11 @@ lint:
         - agent/v1/agent.proto # We want our naming in this file to be different
     PACKAGE_DIRECTORY_MATCH: # Address these warnings during API restructuring
       - management/v1/agent/agent.proto
-      - management/v1/alerting/alerting.proto
-      - management/v1/alerting/params.proto
       - management/v1/azure/azure.proto
```

Comment:
I meant the ones that left, azure, agent, services and node

Reply:
I wasn't sure what to do with those remaining.

Let's consider an example. We have an `agent.proto` in `api/agent/v1/agent.proto`, which provides an `AgentService` that allows to establishe a two-way communication channel between `pmm-agent` and `pmm-managed`. We also have another `agent.proto` in `api/management/v1/agent.proto`, which is still part of the `management` API and should stay that IMO. 

The ticket was about moving those non-management APIs away from the `management` folder since it was confusing.

Azure also seems to belong to the `management` API, this is why I didn't touch it.

Reply:
let's include them into management